vision/classification_and_detection/python/main.py

```diff
@@ -532,8 +533,9 @@ def process_latencies(latencies_ns):
         settings.server_target_latency_ns = int(args.max_latency * NANO_SEC)
         settings.multi_stream_target_latency_ns = int(args.max_latency * NANO_SEC)
 
+    performance_sample_count = args.performance_sample_count if args.performance_sample_count else count
     sut = lg.ConstructSUT(issue_queries, flush_queries, process_latencies)
-    qsl = lg.ConstructQSL(count, min(count, 500), ds.load_query_samples, ds.unload_query_samples)
+    qsl = lg.ConstructQSL(performance_sample_count, min(performance_sample_count, 200), ds.load_query_samples, ds.unload_query_samples)
```

Comment:
```
uintptr_t ConstructQSL(
    size_t total_sample_count, size_t performance_sample_count,
...
```
Shouldn't this be `lg.ConstructQSL(count, performance_sample_count, ...)`? Also, why do we clamp performance_sample_count at 200? Thanks